cloudslang-tests/src/test/resources/yaml/simple_flow_one_liner_binding_check.yaml

```diff
@@ -19,4 +19,4 @@ flow:
   workflow:
     - validate_binding:
         do:
-          ops.compare_values: eq_str_bla = 'bla', eq_int_22 = int_value, eq_bool_true = true
+          ops.compare_values: eq_str_bla = ${ 'bla' }, eq_int_22 = ${ int_value }, eq_bool_true = ${ true }
```

Comment:
Shouldn't the `bla` arg have a string const value?
And the `eq_bool_true ` have a true boolean value?